source/i18n/en-US.ts

```diff
@@ -1,4 +1,75 @@
+import { textJoin } from 'mobx-i18n';
+
 export default {
```

Comment:
```suggestion
import enUS_2017 from '../page/2017/translation/en-US';

export default {
  ...enUS_2017,
```
以此类推，这样就只需要初始化一个 `TranslationModel` 就可以了，还能复用不同页面上可能会相同的一些文本。